src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "User Actions"
+metaDescription: "Understand where your users are clicking, scrolling, and interacting with your web app using User Actions in New Relic Browser."
+freshnessValidatedDate: 2024-04-23
+---
+
+User Actions in New Relic Browser provide insights into how users interact with your web app. You can see where users are clicking, scrolling, and interacting with your web app, and use this data to optimize your app's performance and user experience.
+
+* **Understand user behavior**: Identify the most used features and areas of your web app, and how users link actions together.
+* **Improving the user experience**: Discover areas of your web app that are causing users frustration. For example, you might see that users are getting are stuck on a particular page element or form field.
+* **Identifying issues in key user journeys**: Identify areas of your web app that are causing performance problems. For example, if key user actions are leading to errors, you'd prioritize fixing this critical user journey.
+
+
+## How user actions work [#how-it-works]
+
+When a user loads your web page, the New Relic Browser Agent sets up event listeners for key interaction events. When those events are observed, User Action events are generated and harvested to New Relic.
+
+User Actions currently encapsulate the following events:
+
+* auxclick
+* blur (window only)
+* click
+* copy
+* focus (window only)
+* keydown
+* paste
+* scrollend
```

Comment:
Not sure if we wanted to call out that scrollend doesn't consider the DOM target or it's too low-level for this doc.

Reply:
I think that may be too low-level here